app/components/point.jsx

```diff
@@ -206,13 +225,13 @@ const useStylesFromThemeFunction = createUseStyles(theme => ({
   // shared styling
   sharedBorderStyle: {
     borderRadius: '0.9375rem',
-    boxShadow: theme.boxShadow,
+    boxShadow: '3px 3px 5px rgba(0, 0, 0, 0.05)',
```

Comment:
please put this in theme, give it a name like pointShadow and use rem instead of px.